api/net/Areas/Helpers/ReportHelper.cs

```diff
@@ -64,6 +64,29 @@ public class ReportHelper : IReportHelper
     #endregion
 
     #region Methods
+    /// <summary>
+    /// Order the content based on the session field.
+    /// </summary>
+    /// <param name="content"></param>
+    /// <param name="orderByField"></param>
+    /// <returns>Ordered Content</returns>
+    /// <exception cref="InvalidOperationException"></exception>
+    public static ContentModel[] OrderBySectionField(ContentModel[] content, string orderByField)
```

Comment:
You will need to add this logic to the `ReportingManager.cs` otherwise when the report is generated and sent by email it won't sort by the field.  If you can come up with a clean way to centralize this logic so that it isn't duplicated it would be create.  Currently the `ReportHelper` has a bit of duplication that will cause maintenance issues going forward.